src/temp/CarbonCapture/BarGraph.tsx

```diff
@@ -0,0 +1,110 @@
+import style from './CarbonCapture.module.scss';
+import { useTranslation } from 'next-i18next';
+import { getFormattedNumber } from '../../utils/getFormattedNumber';
+
+interface CO2BarGraphProps {
+  beforeIntervation: number;
+  byProject: number;
+  sitePotential: number;
+}
+
+export const CO2BarGraph = ({
+  beforeIntervation,
+  byProject,
+  sitePotential,
+}: CO2BarGraphProps) => {
+  const beforeIntervationIndicator = (
```

Comment:
The functions `beforeIntervationIndicator` and `byProjectIndicator` could be combined in a single function